docs/api/api_blueprint.apib

```diff
@@ -16235,17 +16235,15 @@ Access Codes related to the events.
 
 | Parameter | Description | Type | Required |
 |:----------|-------------|------|----------|
-| `code`  | Access Code | string | - |
+| `code`  | Access Code | string | yes |
 | `access-url` | URL of the access code | string | - |
-| `type` | Amount or Percentage | string | - |
 | `is-active` | If the access code is active | boolean | - |
 | `tickets-number` | Tickets Number for Access Code | integer (>=0) | - |
 | `min-quantity` | Minimum number of tickets for Access Code | integer (>=0) | - |
 | `max-quantity` | Maximum number of tickets for Access Code | integer (>=0) | - |
 | `valid-from` | Access Code valid from | ISO 8601 (tz-aware) | - |
 | `valid-till` | Access Code valid till | ISO 8601 (tz-aware) | - |
 | `created-at` | Access Code created at | ISO 8601 (tz-aware) | - |
-| `tickets` | Event or Ticket | String | - |
 | `used-for` | Access Code used for | String | - |
```

Comment:
@niranjan94 One query. Is the behaviour of access code supposed to be similar to discount code ? Will there be two types of access codes i.e. one created by the super_admin and used for all the events and second created by the event organiser and only used inside that event ?

Reply:
No. There is only one type of access code. `created by the event organiser and only used inside that event`

Reply:
Alright, then we don't need the variable ```used-for``` as well.